pkg/apis/devconsole/v1alpha1/zz_generated.openapi.go

```diff
@@ -13,9 +13,9 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"github.com/redhat-developer/devconsole-api/pkg/apis/devconsole/v1alpha1.GitSource":       schema_pkg_apis_devconsole_v1alpha1_GitSource(ref),
-		"github.com/redhat-developer/devconsole-api/pkg/apis/devconsole/v1alpha1.GitSourceSpec":   schema_pkg_apis_devconsole_v1alpha1_GitSourceSpec(ref),
-		"github.com/redhat-developer/devconsole-api/pkg/apis/devconsole/v1alpha1.GitSourceStatus": schema_pkg_apis_devconsole_v1alpha1_GitSourceStatus(ref),
+		"github.com/redhat-developer/git-service/pkg/apis/devconsole/v1alpha1.GitSource":       schema_pkg_apis_devconsole_v1alpha1_GitSource(ref),
```

Comment:
Wrong package.

Reply:
When I regenerated the API within the `devconsole-operator` repo, then this change disappeared. In addition, the component API is missing there as well. 
Btw, @sbose78 why is the `component_types.go` (and the whole [apis](https://github.com/redhat-developer/devconsole-operator/tree/master/pkg/apis) directory) present in the devconsole-operator repo at all. And with different content than in the `api` repo? It brings to the whole generate/copy/paste flow additional complications.

Reply:
It will be moved out in the next couple of days :)